canonical-websites/www.ubuntu.com
# 470 new organisation page

## Done

Remove `/desktop/education`, `/desktop/enterprise`, and `/desktop/government` and redirect to `/desktop/ubuntu-in-organisations`

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: <http://0.0.0.0:8001/desktop/education>
- Check that the page redirects to `/desktop/ubuntu-in-organisations`
- View the site locally in your web browser at: <http://0.0.0.0:8001/desktop/enterprise>
- Check that the page redirects to `/desktop/ubuntu-in-organisations`
- View the site locally in your web browser at: <http://0.0.0.0:8001/desktop/government>
- Check that the page redirects to `/desktop/ubuntu-in-organisations`

## Issue / Card

Fixes #474
Fixes #2891 
Fixes #2892
